src/cubitpy/cubitpy_types.py

```diff
@@ -264,15 +264,21 @@ def get_default_four_c_description(self):
             or self == self.tet10
             or self == self.wedge6
         ):
-            return "KINEM nonlinear"
+            return {"KINEM": "nonlinear"}
         elif self == self.hex8sh:
-            return "KINEM nonlinear EAS none ANS none THICKDIR auto"
+            return {
+                "KINEM": "nonlinear",
+                "EAS": "none",
+                "ANS": "none",
+                "THICKDIR": "auto",
+            }
```

Comment:
This will have to be changed once we tackle the solid shell implementation.

Reply:
This should fix the solid shell creation (the reference files will have to be adapted)
```suggestion
            return {
                "KINEM": "nonlinear",
                "TECH": "shell_eas_ans"
            }
```